### DISCUSSION TITLE
Align gallery items - height alignment?

### DISCUSSION THREAD

**Discussion by joush007** [nunocoracao/blowfish#642] [Q&A]
I've been experimenting with the blowfish theme for a bit now, and I've come across a small issue. I haven't seen any other discussions or issues about this, but if there is, sorry I can close this. It has to do with the height alignment in the gallery of posts/cards or whatever you'd like to call them (as seen in [https://blowfish.page/docs/](https://blowfish.page/docs/) with the Partials post next to the other posts).

I'm unsure if there is a fix for this except for changing the actual css behind the theme, if that's the only way, I can give that a shot with editing the css, otherwise I'm just wondering if there's any other ways to fix that?

**Comment by nunocoracao** [comment]
Hello @joush007 could you provide a screenshot example of what you mean?

**Reply by joush007** [reply]
Yep, sorry I forgot to earlier:
![image](https://user-images.githubusercontent.com/84158931/229270860-21513842-54db-4b94-9bd9-a124526d18b6.jpeg)

Just the height difference due to tags and such or names of pages tend to create the height differences to fit it all in I'd assume

**Comment by nunocoracao** [comment]
Not at this stage. Could be an improvement even though I am not really sure if just extending the shorter ones will do the trick
